rust-runtime/aws-smithy-http-server-python/examples/pokemon_service.py

```diff
@@ -78,6 +79,10 @@ def value(self) -> int:
 #   https://docs.python.org/3/library/multiprocessing.html#sharing-state-between-processes
 @dataclass
 class Context:
+    # Inject Lambda context if service is running on Lambda
+    # NOTE: All the values that will be injected by the framework should be wrapped with `Optional`
+    lambda_ctx: Optional[LambdaContext] = None
```

Comment:
Just for my understanding: if the `lambda_ctx` of type `Optional[LambdaContext]` is not defined here, the injection will be just avoided right?